src/huggingface_hub/repocard_data.py

```diff
@@ -113,9 +115,12 @@ class EvalResult:
     # Example for `bleu`: max_order: 4
     metric_args: Optional[Dict[str, Any]] = None
 
-    # If true, indicates that evaluation was generated by Hugging Face (vs. self-reported).
+    # Automatically computed, do not set. Dynamically overridden by Hugging Face in API calls to indicate if evaluation was verified by Hugging Face.
     verified: Optional[bool] = None
 
+    # Generated by Hugging Face to verify the results are valid.
+    verifyToken: Optional[str] = None
```

Comment:
I use camelCase here because that's what's implemented on the Hub side: https://github.com/huggingface/moon-landing/pull/4037/files#diff-e9eafbf94fe033bdaf22611fa3eab89f02aa8776685889dd5cace85d7ed82257R82

Reply:
@lewtun Sorry to ask for a last change but would it be possible to use snake_case here ? I think keeping consistency within the `hfh` parameters is more important than keeping consistency with the server naming.

This would require to change both the getter and the setter in
```py
# in model_index_to_eval_results
verify_token = metric.get("verifyToken")
(...)
verify_token=verify_token,
```
and
```py
# in eval_results_to_model_index
"verifyToken": result.verify_token,
```

Reply:
Thanks, I agree snake_case is nicer :)

Fixed in [f0d1cc4](https://github.com/huggingface/huggingface_hub/pull/1142/commits/f0d1cc4399c4959aa016d77109577db0fef2b718) and [6995d2a](https://github.com/huggingface/huggingface_hub/pull/1142/commits/6995d2aa2dfdb7a4c3230002fd72412857d0a188)